Examples/TicTacToe/Sources/Core/TwoFactorCore.swift

```diff
@@ -56,6 +60,7 @@ public let twoFactorReducer = Reducer<TwoFactorState, TwoFactorAction, TwoFactor
       .receive(on: environment.mainQueue)
       .catchToEffect()
       .map(TwoFactorAction.twoFactorResponse)
+      .cancellable(id: TwoFactorTearDownToken())
```

Comment:
Is it possible to have multiple cancellation tokens? For instance, one unique token that could be used for each type of effect in a reducer, and another "general" token that could be used to cancel everything? So for each effect you'd have to add a cancellable for both tokens.

That may not be much nicer than the enum approach you mentioned in another comment, but I'm just wondering if it's feasible.